test/sewing_test.c

```diff
@@ -158,17 +158,12 @@ void Sew_Test_Set_hwloc_thread_affinity
                             !hwloc_set_thread_cpubind
                             (
                                 t
-                                , threads[threads_left - 1]
+                                , threads[threads_left - 1].handle
```

Comment:
Yeah, I guess the only alternative would be to make it a struct in linux as well, but I have no idea what other issues that would cause.